apis/projectcontour/v1/httpproxy.go

```diff
@@ -216,7 +216,9 @@ type TLS struct {
 	// If specified, the named secret must contain a matching certificate
 	// for the virtual host's FQDN.
 	SecretName string `json:"secretName,omitempty"`
-	// Minimum TLS version this vhost should negotiate
+	// MinimumProtocolVersion is the minimum TLS version this vhost should
+	// negotiate. Valid options are `1.2` (default) and `1.3`. Any other value
+	// defaults to TLS 1.2.
```

Comment:
> Any other value defaults to TLS 1.2.

Do we get a warning or anything if this case is hit?

Reply:
Nope. We can change that behavior if desired; I just left it the way it already was but not sure if there was a reason for it or not.

Reply:
This is documenting the current behavior, I don't believe there's any reason for people to run the massively insecure TLS 1.1, if there is, they can come and ask.

Reply:
@youngnick no my point is if someone sets an invalid option, say they make the value `99.99`, Contour will change that to `1.2`. I'm cool with it in this PR since that's how it works, but there's no indication to a user, that they 1. set an invalid value and 2. Contour saw that and changed it to the default.